docs/specs/config.yml

```diff
@@ -167,20 +173,28 @@ inputs:
     routes:
       docs/a/: output
       docs/a/b.md: outputB
-  docs/a/b.md:
+      docs/a/c.gif: outputC
+  docs/a/b.md: |
+    [link](c.gif)
+  docs/a/c.gif:
 outputs:
   outputB/b.json:
+  outputC/c.gif:
   build.manifest:
 ---
 # latter routes rule takes precedence - 2
 inputs:
   docfx.yml: |
     routes:
       docs/a/b.md: outputB
+      docs/a/c.gif: outputC
```

Comment:
remove this line, to check `docs/a` should include `docs/a/c.gif`

Reply:
this line doesn't influent the check, but can also check the routes priority in resource

---
In reply to: [203285427](https://github.com/dotnet/docfx/pull/3092#discussion_r203285427) [](ancestors = 203285427)